vassal-app/src/main/resources/VASSAL/i18n/VASSAL_da.properties

```diff
@@ -1023,6 +1023,7 @@ PlayerRoster.join_another_side=Skift til en anden side
 PlayerRoster.give_up_position=Vil du opgive din position som %1$s?
 PlayerRoster.join_game_as=Join spillet p� hvilken side?
 PlayerRoster.switch_sides=Din nuv�rende side er %1$s. Skift til hvilken side?
+PlayerRoster.switch_sides2=%1$s er nu ikke tilg�ngelig. Din nuv�rende side er %2$s. Skift til hvilken side?
```

Comment:
Normally, unless you are very confident (native language speaker level) of the translation, don't worry about adding non-English translations. This will be (is supposed to be) done 'later' (!) by a native speaker.

Reply:
Point taken. I felt I needed at least one translation for testing. If this thing goes live I plan to highlight the changes so hopefully at least the recent translators (Danish for example) will check the Google's work.